src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -888,11 +896,9 @@ public override void Accept(OperationVisitor visitor)
 
     internal partial class BoundBaseReference : IInstanceReferenceExpression
     {
-        bool IInstanceReferenceExpression.IsExplicit => this.Syntax.Kind() == SyntaxKind.BaseExpression;
-
-        IParameterSymbol IParameterReferenceExpression.Parameter => (IParameterSymbol)this.ExpressionSymbol;
+        InstanceReferenceKind IInstanceReferenceExpression.InstanceReferenceKind => InstanceReferenceKind.BaseClass;
```

Comment:
@nguerrera is correct that the this/base/class distinction can be inferred. Is the inference too subtle for analyzer authors?

What are the other possibilities in IL? Can they be generated from C# or VB?